shark_turbine/kernel/wave/distribution_symbols.py

```diff
@@ -0,0 +1,9 @@
+import shark_turbine.kernel.lang as tkl
+
+WORKGROUP_0 = tkl.sym.WG0
```

Comment:
Does WG0, WG1, Wg2 map to WG_X, WG_Y, WG_Z ?

Reply:
WG[ i ]  maps to stream dialect dispatch workgroup id [ i ] . I am not sure how stream dialect maps 0, 1, 2 to x,y,z. I think it maybe reversed?